am-lib/src/test/java/uk/gov/hmcts/reform/amlib/internal/FilterServiceTest.java

```diff
@@ -368,6 +383,81 @@ void itShouldBePossibleToShowEverythingExceptTopLevelObjectWithLeafLevelExceptio
         );
     }
 
+    @Test
+    void itShouldBePossibleToShowEverythingExceptTopLevelObjectWithMultipleLeafLevelExceptions() throws IOException {
+        JsonNode inputJson = mapper.readTree(ClassLoader.getSystemResource("FilterServiceResources/input.json"));
+
+        Map<JsonPointer, Set<Permission>> attributePermissions = ImmutableMap.of(
+            JsonPointer.valueOf(""), ImmutableSet.of(READ),
+            JsonPointer.valueOf("/claimant"), ImmutableSet.of(CREATE),
```

Comment:
Do we need randomize this?

Reply:
These unit tests use a complex data structure which would be arduous to randomise. This method is also covered by integrations tests which do use randomised data, so I don't think it's necessary to randomise the data here

Reply:
👍 agreed with @shashank-rastogi that randomisation is not necessary here